### PR TITLE
Installations: Installing Google chrome on Linux Fix code block formatting for deb file installation

### PR DESCRIPTION
**Pull request by emberavenge**
## Because

Corrected the section of code that instructs users on how to install the Google Chrome deb file. It may cause issues for those who copy and paste, as well as for those who type the commands. 

P.S I have ticked the allow edits by maintainers in case I might've missed something, thanks!

## This PR

- Grammar/Formatting error

## Issue

Closes #XXXXX

## Pull Request Requirements

-   [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
-   [x] The title of this PR follows the `location of change: brief description of change` format, e.g. `Intro to HTML and CSS lesson: Fix link text`
-   [x] The `Because` section summarizes the reason for this PR
-   [x] The `This PR` section has a bullet point list describing the changes in this PR
-   [ ] If this PR addresses an open issue, it is linked in the `Issue` section
-   [ ] If any lesson files are included in this PR, they have been previewed with the [Markdown preview tool](https://www.theodinproject.com/lessons/preview) to ensure it is formatted correctly
-   [ ] If any lesson files are included in this PR, they follow the [Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

### CODE REVIEW

File: foundations/installations/installations.md [hunk 1]
```diff
@@ -298,9 +298,8 @@ wget https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb
 
 - Enter the following command in your terminal to install **Google Chrome** `.deb` package
 
-       ~~~bash
        sudo apt install ./google-chrome-stable_current_amd64.deb
-       ~~~
```
Comment (rlmoser99) [inline foundations/installations/installations.md:303]: If you use the markdown preview tool that is listed in the checklist, you can see the effects of your changes. 
https://www.theodinproject.com/lessons/preview

I believe the intention is to have it in a code block, which this change removes. The issue with the code block is that there were too many spaces before this line.

     ```bash
     sudo apt install ./google-chrome-stable_current_amd64.deb
     ```

File: foundations/installations/installations.md [hunk 2]
```diff
@@ -298,9 +298,8 @@ wget https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb
...
+
```
Comment (rlmoser99) [inline foundations/installations/installations.md:302]: If you use the markdown preview tool that is listed in the checklist, you can see the effects of your changes. 
https://www.theodinproject.com/lessons/preview

I believe the intention is to have it in a code block, which this change removes. The issue with the code block is that there were too many spaces before this line.

```suggestion
    ```bash
    sudo apt install ./google-chrome-stable_current_amd64.deb
    ```
```

**Review by rlmoser99 — APPROVED** [review]
Thank you for bringing to our attention and for taking the time to make a PR. Working with our markdown parser (Kramdown) takes a bit of trial and error to make sure it is doing what you want.

**Comment by emberavenge** [comment]
Glad I could be at least of some help, thanks for the suggestions, next time i'll use the markdown parser.

**Pull request merged**